src/main/java/com/easypost/model/RateDeserializer.java

```diff
@@ -131,6 +139,8 @@ public Rate deserialize(final JsonElement json, final Type typeOfT, final JsonDe
         rate.setEstDeliveryDays(estDeliveryDays);
         rate.setShipmentId(shipmentID);
         rate.setCarrierAccountId(jo.get("carrier_account_id").getAsString());
+        rate.setCarbonOffset(carbonOffset);
+        rate.setBillingType(jo.get("billing_type").getAsString());
```

Comment:
DEVELOPER NOTE: Went ahead and fixed `billing_type`, was not being serialized as well. Must have slipped through the cracks a long time ago.